index.bs

```diff
@@ -3967,6 +3987,26 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Capability Enumeration (enum <dfn enum>ClientCapability</dfn>) ### {#enum-clientCapability}
+
+<xmp class="idl">
+    enum ClientCapability {
+        "hybrid-transport",
+        "client-pin-entry"
+    };
+</xmp>
+
+This enumeration defines a limited set of client capabilities which a [=[WRP]=] may evaluate to offer certain workflows and experiences to users.
+
+Note: The {{ClientCapability}} enumeration is deliberately not referenced, see [[#sct-domstring-backwards-compatibility]].
+
+<div dfn-type="enum-value" dfn-for="ClientCapability">
+    :   <dfn>hybrid-transport</dfn>
+    ::  The [=WebAuthn Client=] supports the {{AuthenticatorTransport/hybrid}} transport.
```

Comment:
I wonder if "supports" is strong enough a signal. Yes, in absolute terms Chrome "supports" hybrid even when it lacks the OS-level Bluetooth permission. But in relative terms Chrome cannot "support" hybrid because it lacks access to Bluetooth. Which nuance is this intending to convey to an RP?

Reply:
This is more of an "absolute support": a client can confirm all the pieces are there for use of hybrid, even though it may require user remediation to make it possible to successfully complete a ceremony over hybrid.